### PR TITLE
 #166 add event type message template

### PR DESCRIPTION
**Pull request by aleevas**
This should add an ability to configure the messages from RNG

### CODE REVIEW

File: src/EventMeta.php [hunk 1]
```diff
@@ -608,4 +610,64 @@ protected function entityTypeHasBundles($entity_type_id) {
...
+   * {@inheritdoc}
+   */
+  function createDefaultEventMessages() {
+    // Get Default messages for this Event type.
+    $default_messages = $this->getEventType()->getDefaultMessages();
+    if ($default_messages) {
+      foreach ($default_messages as $default_message) {
+        // Create Event Messages from Default Messages.
+        /** @var \Drupal\courier\Service\CourierManagerInterface $courier_manager */
+        $courier_manager = \Drupal::service('courier.manager');
```
Comment (aleevas) [inline src/EventMeta.php]: Why you use these services via static method instead using Dependency Injection?

File: src/Form/EventTypeDefaultMessagesListForm.php [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+    $event_type->setDefaultMessages($default_messages)->save();
+
+    // Site cache needs to be cleared after enabling this setting as there are
+    // issue regarding caching.
+    // For some reason actions access is not reset if pages are rendered with no
+    // access/viability.
+    Cache::invalidateTags(['rendered']);
+
+    drupal_set_message($this->t('Event type access defaults saved.'));
```
Comment (Sergey-Orlov) [inline src/Form/EventTypeDefaultMessagesListForm.php]: Bad message text (copy-paste). Should be updated to have proper text.

File: src/EventMeta.php (hunk 1 above)
Comment (Sergey-Orlov) [reply src/EventMeta.php]: fixed, plz re-check

**Comment by aleevas** [comment]
Looks good to me

**Pull request merged**